bugalink-backend/users/models.py

```diff
@@ -1,6 +1,20 @@
 from django.contrib.auth.models import AbstractBaseUser, BaseUserManager
+from django.core.files.storage import FileSystemStorage
 from django.db import models
 
+def user_directory_path(instance, filename):
+    # print(instance.user)
+    return "users/{0}/{1}".format(instance.id, filename)
+
+class OverwriteStorage(FileSystemStorage):
+    def get_available_name(self, name, max_length=None):
+        original_name=self.url(name).split(".")[0]
+        new_fileName = name.split(".")[0]
```

Comment:
you are mixing snake_case and camelCase in the same variable, please follow the conventions